libraries/AP_Camera/examples/tracking.py

```diff
@@ -0,0 +1,161 @@
+import cv2
+import threading
+from pymavlink import mavutil
+import math 
+from ultralytics import YOLO
+
+master = mavutil.mavlink_connection('127.0.0.1:14560')
```

Comment:
we should allow the caller to specify the video source as an argument to the script.

Reply:
instructions on how to install would be good.  E.g. what packages are required and how can they be installed? (e.g. pip)